music21/musicxml/xmlToM21.py

```diff
@@ -6217,15 +6155,14 @@ def testStaffLayoutMore(self):
                 sizes.append(s.staffSize)
         self.assertEqual(sizes, [80.0, 120.0, 80.0])
 
-
     def testCountDynamics(self):
         '''
         good test of both dynamics and a partstaff...
         '''
-        from music21 import  corpus, converter
+        from music21 import corpus, converter
         # NB using getWorkList to change format to oldmusicxml as necessary for testing.
         c = converter.parse(corpus.corpora.CoreCorpus().getWorkList(
-                                                    'schoenberg/opus19/movement2.mxl')[0],
+            'schoenberg/opus19/movement2.mxl')[0],
```

Comment:
having the filename, which is inside the 'getWorkList()' call unindented farther than the "format='musicxml'" obscures the nested relationship.

Reply:
made a few changes to make these lines consistent and hopefully more readable.